backend/oidc/jobs/hourly/clear_user_sessions.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+from django_extensions.management.jobs import HourlyJob
+
+from oidc.models import EAuthorizationProfile, OIDCProfile
+from oidc.services import clear_eauthorization_profiles, clear_oidc_profiles
+
+
+class Job(HourlyJob):
```

Comment:
Need to set up cron so that the hourly jobs are executed.
How should it be set up? Can the similar setup be used as was used in Venepaikka to execute the jobs in a separate pod?

Reply:
Yea, this is true. Have to investigate how it should be done. Venepaikka could be a good example, I'll check it out.